nix/third-party/gRPC-haskell/core/BUILD.bazel

```diff
@@ -11,24 +11,12 @@ load(
 )
 load("//bazel_tools:fat_cc_library.bzl", "fat_cc_library")
 
-c2hs_suite(
+da_haskell_library(
```

Comment:
Can we have a comment and fixme that we generated the .hs files manually here. (I can't think of a better place to put it)

Reply:
Yes, I've added a comment.